setup/db/db/schema-490to491.sql

```diff
@@ -0,0 +1,4 @@
+
+LOCK TABLES `vm_template` WRITE;
+INSERT INTO `vm_template` VALUES (13,'macchinina-x86_64-xen','Macchinina (64-bit) no GUI (XenServer)','9511a500-620b-11e6-843c-00505635944c',1,1,'BUILTIN',0,64,'http://dl.openvm.eu/cloudstack/macchinina/x86_64/macchinina-xen.vhd.bz2','VHD','2016-08-14 16:11:00',NULL,1,'30985504bc31bf0cd3b9d2c6ca7944d3','Macchinina(64-bit) no GUI (XenServer)',0,0,99,1,0,1,1,'XenServer',NULL,NULL,0,NULL,'Active',0,NULL,0),(14,'macchinina-x86_64','Macchinina (64-bit) no GUI (KVM)','9512a500-620b-11e6-843c-00505635944c',1,1,'BUILTIN',0,64,'http://dl.openvm.eu/cloudstack/macchinina/x86_64/macchinina-kvm.qcow2.bz2','QCOW2','2016-08-14 16:11:02',NULL,1,'f50acb3a4387019b473d0f25a42bf06e','Macchinina (64-bit) no GUI (KVM)',0,0,99,1,0,1,1,'KVM',NULL,NULL,0,NULL,'Active',0,NULL,0),(15,'macchinina-x64','Macchinina (64-bit) no GUI (vSphere)','9513a500-620b-11e6-843c-00505635944c',1,1,'BUILTIN',0,64,'http://dl.openvm.eu/cloudstack/macchinina/x86_64/macchinina-vmware.ova','OVA','2016-08-14 16:11:05',NULL,1,'f6f881b7f2292948d8494db837fe0f47','Macchinina (64-bit) no GUI (vSphere)',0,0,99,1,0,1,1,'VMware',NULL,NULL,0,NULL,'Active',0,NULL,0);
+UNLOCK TABLES;
```

Comment:
This change will add xen macchinina template as builtin templates to cloudstack. This will ensure that these templates are available by default like the Centos templates.